data/abilities/collection/1f7ff232-ebf8-42bf-a3c4-657855794cfe.yml

```diff
@@ -16,3 +16,5 @@
           name: regex
           property: target.org.email
           script: '[\w\.-]+@[\w\.-]+'
+  consequence: |
```

Comment:
this combo doesn't make sense ;) - the host.user.name doesn't have the target.org.email addresses in this case, they are just found on this user's computer.